extensions/mochi-gg/src/ticker.tsx

```diff
@@ -0,0 +1,150 @@
+import { useEffect, useState } from "react";
+import { formatCurrency, formatPercent } from "./utils/string";
+import _ from "lodash";
+import useDiscord from "./hooks/useDiscord";
+import useWatchList from "./hooks/useWatchList";
+import { MOCHI_PROXY_ENDPOINT } from "./config/cfg";
+
+export default function Main({
+  arguments: props,
+  launchContext,
+}: LaunchProps<{
+  arguments: { token: string };
+  launchContext: { token: string; src: string; disableSearch: boolean };
+}>) {
+  const [interval, setInterval] = useState(7);
+  const [token, setToken] = useState((props?.token || launchContext?.token || "bitcoin").replace(/ /g, "-"));
+  const { user, loginDiscord, logoutDiscord } = useDiscord(false);
+  const { addTokenToWatchlist, watchingMap, removeTokenFromWatchlist } = useWatchList();
+
+  const { data: { data: tokens = [] } = {} } = useFetch<{ markdown?: string } & Record<string, any>>(
+    !launchContext?.disableSearch
+      ? `https://api.mochi.pod.town/api/v1/defi/coins?query=${props?.token || launchContext?.token || "btc"}`
+      : ""
+  );
+
+  const {
+    isLoading,
+    data: { markdown, base_coin: data } = {},
+    error,
+    revalidate,
+  } = useFetch<{ markdown?: string } & Record<string, any>>(
+    `${MOCHI_PROXY_ENDPOINT}/api/ticker?token=${token || "bitcoin"}&time_step=${interval}&currency=usd&theme=${
+      environment.theme
+    }`
+  );
+
+  useEffect(() => {
+    if (error) {
+      showToast({
+        style: Toast.Style.Failure,
+        title: "Something went wrong",
+        message: error.message,
+      });
+    }
+  }, [error]);
+
+  const AddWatchListAction = (item: any) =>
+    user?.id ? (
+      <>
+        <Action
+          title={watchingMap[item?.id] ? "Remove From Watch List" : "Add to Watch List"}
+          onAction={async () => {
+            if (watchingMap[item?.id]) {
+              removeTokenFromWatchlist(item);
+            } else {
+              addTokenToWatchlist(item);
+            }
+          }}
+          // @ts-ignore
+          style={watchingMap[item?.id] ? "destructive" : "regular"}
+          icon={watchingMap[item?.id] ? Icon.Trash : Icon.Plus}
+        />
+        <Action title="Log Out" onAction={logoutDiscord} icon={Icon.Logout} />
+      </>
+    ) : (
+      <Action
+        title={"Connect with Discord"}
+        onAction={async () => {
+          loginDiscord();
+        }}
+        icon={environment.theme === "light" ? "discord-mark-black.png" : "discord-mark-white.png"}
+      />
+    );
+
+  if (!data && error) {
+    return (
+      <List onSearchTextChange={(txt) => setToken(txt)} searchBarPlaceholder="Search for other token" throttle>
+        <List.EmptyView icon={Icon.MagnifyingGlass} title={`"${token}" Not found`} />
+      </List>
+    );
+  }
+
+  return (
+    <Detail
+      isLoading={!markdown || !data || isLoading}
+      markdown={markdown || ""}
+      navigationTitle={`Exchange rates ${_.upperCase(data?.symbol || token)}/USD • Coingecko | Mochi`}
```

Comment:
Don't change the navigationTitle in the root command - it will be automatically set to the command name. Use navigationTitle only in nested screens to provide additional context.